wrapanapi/systems/lenovo.py

```diff
@@ -186,11 +190,10 @@ def get_server_health_state(self, server_name):
         elif str(server['cmmHealthState'].lower()) in self.HEALTH_CRITICAL:
             return "Critical"
         else:
-            return "Unknow"
+            return "Unknown"
```

Comment:
In general, it would be preferable to return an enum here, doesn't need to be included in this PR though.

Consider changing the `HEALTH_CRITICAL` tuple to be called `HEALTH_CRITICAL_STATES`, and keep class attrs like `HEALTH_CRITICAL` to be a single string of the returned state.

This allows for callers to look for a health state matching some programmatic value for example: `assert mgmt.get_server_health_state() == mgmt.HEALTH_VALID` versus `assert mgmt.get_server_health_state() == 'OopsTypo'`

Reply:
For further examples, see how the `wrapanapi.entities.VmState` class of enums is used for controlling VM power.

Reply:
I guess I'll do it when I refactor the Lenovo Servers as well, which I hope to do soon.